core/src/cluster_nodes.rs

```diff
@@ -480,11 +484,46 @@ pub fn make_test_cluster<R: Rng>(
     (nodes, stakes, cluster_info)
 }
 
+pub(crate) fn get_data_plane_fanout(shred_slot: Slot, root_bank: &Bank) -> usize {
+    if !enable_turbine_fanout_experiments(shred_slot, root_bank) {
+        return DATA_PLANE_FANOUT;
+    }
+    // Allocate ~2% of slots to fanout experiment.
+    match shred_slot % 359 {
```

Comment:
why 359 instead of 350?

Reply:
Picked some prime number to avoid possible coincidental patterns.
Probably doesn't matter much.